src/malloy/data/bigquery/bq_connection.py

```diff
@@ -51,15 +51,15 @@ def get_schema_for_tables(self, tables: Sequence[(str, str)]):
     for (key, table) in tables:
       schema["schemas"][key] = self._to_struct_def(
           table,
-          bigquery.Client(self._client_options).get_table(table).schema)
+          bigquery.Client(**self._client_options).get_table(table).schema)
```

Comment:
I was going to add typing in here but I couldn't quickly find easy types to import for the bigquery.Client constructor.

Also, w/out an easily fakeable dependency wrapper to inject for the actual `bigquery.Client` object, writing tests for this felt bigger than I'm ready to contribute.